_faq/installation_and_deployment/no-external-ip.md

```diff
@@ -0,0 +1,30 @@
+---
+title: How to deploy Forseti without external IPs?
+order: 1
+---
+{::options auto_ids="false" /}
+Please follow the following steps to install Forseti with internal IP only:
+
+* Prerequisites
+    * Make sure the host subnet has [Google Private IP Access ](https://cloud.google.com/vpc/docs/configure-private-google-access) turned on.
+    * Setup [Cloud NAT](https://cloud.google.com/nat/docs/using-nat) with a bastion host.
+* Install Forseti
+* Configure Cloud SQL
+    * enable internal IP
+    * remove the external IP
+* Remove external IPs from the Forseti VMs
+* Firewall Rules: Depending on how your environment is setup, you'll need to modify the firewall rules accordingly to allow SSH communication between the bastion host and Forseti VMs. A few suggestions:
+    * Make sure the bastion host can access forseti VMs
+    * Modify firewall rules created by Forseti installer limiting them to the subnet instead of 0.0.0.0/0
+    * Limit Forseti egress to ports needed for pip and os updates
+
+You are all set! To verify it works, access the forseti VMs by [connecting through a bastion host](https://cloud.google.com/compute/docs/instances/connecting-advanced#bastion_host).
+
+A few common causes if you cannot access Forseti VMs from the bastion host:
+* Your account does not have the appropriate permission
+    * You might need to run:
+        ```bash
+        gcloud auth login
```

Comment:
Which project do I need to be auth'd against?  The Forseti project?  If so, we should mention either the Forseti project or the project of the bastion host.

Reply:
When executing gcloud auth login, there was no prompt to select a project (unless I missed it). reference: https://cloud.google.com/sdk/gcloud/reference/auth/login.

Reply:
I'm mentioning gcloud auth login because sometimes "Permission Denied" error occurs not because the user account lacks permission, but because the user session in gcloud is out. In that case, gcloud auth login re-authenticate the user and creates a new session, so that their credentials can be properly accessed and evaluated by gcloud.